### PR TITLE
Update CentOS packaging details to 1.8 

### PR DESCRIPTION
**Pull request by joel-simpson**
Update Installation instructions (update url for packages from 1.7 to 1.8)
Update the Upgrade instructions (update the command to go from 1.7 to 1.8)

connected to https://github.com/archivematica/Issues/issues/203
(This PR only partially resolves that issue as it only covers CentOS. Another PR will be required to update details for Ubuntu.)

### CODE REVIEW

File: admin-manual/installation-setup/installation/install-centos.rst [hunk 1]
```diff
@@ -69,24 +69,24 @@ Installation instructions
    * Archivematica:
 
    Use these commands to install the repositories:
-
+
```
Comment (mamedin) [inline admin-manual/installation-setup/installation/install-centos.rst]: I think it is not so important as python code, but an empty line has been replaced by a line with a tab (or several blank spaces). Same for lines 74 and 83

**Review by mamedin — CHANGES_REQUESTED** [review]
It is about CentOS install and upgrade, so PR title and commit messages should mention it is only for CentOS.

File: admin-manual/installation-setup/installation/install-centos.rst (hunk 1 above)
Comment (joel-simpson) [reply admin-manual/installation-setup/installation/install-centos.rst]: I updated to fix these so they have empty lines as before.

**Review by mamedin — CHANGES_REQUESTED** [review]
Sorry, but lines 74 and 83 still have blank spaces :(

**Comment by joel-simpson** [comment]
sorry @mamedin I didn't check properly the first time. I think I've fixed it properly this time.

**Review by mamedin — APPROVED** [review]
LGTM

**Review by sallain — APPROVED** [review]

**Comment by sallain** [comment]
Merged!

**Pull request closed without merging**